### PR TITLE
RE2022-243: raise if NA value exists

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: src/loaders/genome_collection/parse_PMI_biolog_data.py [hunk 1]
```diff
@@ -116,9 +116,9 @@ def _read_biolog_data(biolog_data_file: Path) -> pd.DataFrame:
     data_df = data_df.drop(index='BT03 A(10/1)')
     data_df = data_df.rename(index={'BT03 B(11/18)': 'BT03'})
 
-    # Remove strain designation if cell value is NaN
-    # Two NaN values are present in the data file for 'YR530' and 'CF286' in the row labeled '3-Hydroxy 2-Butanone'.
-    data_df.dropna(axis=0, inplace=True)
+    # Check for NA values in the DataFrame
+    if data_df.isna().any(axis=1).any():
+        raise ValueError(f"DataFrame contains NA values:\n{data_df[data_df.isna().any(axis=1)]}")
```
[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/genome_collection/parse_PMI_biolog_data.py]: @MrCreosote It prints the rows with NA values.

[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/loaders/genome_collection/parse_PMI_biolog_data.py]: The entire row? Can you give me an example?

[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/genome_collection/parse_PMI_biolog_data.py]: yea. The entire row.

[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/genome_collection/parse_PMI_biolog_data.py]: `ValueError: Biolog data has missing values:
strain designation Negative Control  ...                        growth_media
1                                    ...                                    
BC15                              0  ...                  MOPS minimal media
CF313                             0  ...                  MOPS minimal media
YR530                             0  ...  MOPS minimal media + 0.41uM Biotin

[3 rows x 193 columns]`

[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply src/loaders/genome_collection/parse_PMI_biolog_data.py]: I updated it to just print the strain name
e.g. `ValueError: Some of the strains in the Biolog data exhibit missing values: ['BC15', 'CF313', 'YR530']`

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**